doc/usage.md

```diff
@@ -378,3 +378,29 @@ the operator will leave `PriorityClass` empty, issue a warning, and
 continue scheduling scans without a `PriorityClass`.
 
 [1]: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#priorityclass
+
+## Increasing operator's memory or CPU limits
+
+In some cases, the compliance-operator might require more memory than the default
+limits allow. If the operator had been installed via OLM (which is the case when
```

Comment:
LGTM! Just one minor suggestion. 
```suggestion
limits allow. If the operator had been installed via OLM (which is the case when
```
limits allow. If the operator had been installed through OLM (which is the case when